nature/models.py

```diff
@@ -79,7 +161,7 @@ def __str__(self):
 
 class Value(models.Model):
     explanation = models.CharField(_('explanation'), max_length=50, blank=True, null=True, db_column='selite')
-    value_type = models.CharField(_('value type'), max_length=10, blank=True, null=True, db_column='luokka')
+    value = models.CharField(_('value class'), max_length=10, blank=True, null=True, db_column='luokka')
```

Comment:
Maybe`value_class`?

Reply:
FYI, there's a page explaining about the `value_class` field: http://kartta.hel.fi/Applications/ltj/html/linkitetyt_ltj/kantaan_linkitettyja/Arvottamiskriteerit_kasvikohteet_2012.htm

Reply:
The `value` name was taken from the .xls file that I got, not something that I came up with myself :) There is also a reasoning why I changed the verbose name to `value class` in the commit.

Reply:
![image](https://user-images.githubusercontent.com/3242350/42307416-06c08126-803b-11e8-895c-382515857157.png)

Reply:
But yes, we can probably change it to value_class as well. Do you want me to do that? It is a trivial thing to do so.

Reply:
Ah I see. It sounds a bit strange to me, but let's go with it for now.